lib/sidekiq-status/web.rb

```diff
@@ -75,8 +75,11 @@ def has_sort_by?(value)
 
       app.get '/statuses' do
 
-        namespace_jids = Sidekiq.redis{ |conn| conn.keys('sidekiq:status:*') }
-        jids = namespace_jids.map{ |id_namespace| id_namespace.split(':').last }
+        jids = Sidekiq.redis do |conn|
+          conn.scan_each(:match => 'sidekiq:status:*', :count => 100).map do |key|
```

Comment:
Will the count 100 suffice in all cases? What if the keys are more than 100? 
Also can you use the symbols and replace `=>` as 
```ruby
conn.scan_each(match: 'sidekiq:status:*', count: 100)
```

Reply:
Yes, 100 will work for any number of keys. It is the number that is pulled out of redis during each scan. I also ensured that the number of scans does not affect the array, they are all automatically flattened, see below. The default is 10 which I have found to be a little slow and conservative when working with small keys, 100 should not be any problem and will be a bit faster.
```ruby
(dev)> 20.times{|n| Rails.cache.data.set("test:#{n}", n)}
=> 20
(dev)> Rails.cache.data.scan_each(:match => 'test:*', :count => 5).map(&:itself)
=> ["test:4", "test:5", "test:8", "test:15", "test:17", "test:16", "test:1", "test:10", "test:3", "test:18", "test:11", "test:6", "test:0", "test:19", "test:13", "test:9", "test:12", "test:2", "test:7", "test:14"]
```

Will update the syntax and look into that build failure! Thanks!